Fix readme typos

Jennit07 opened the pull request:
few typos on some CHI to anonCHI examples. Added in a sample cohort example

Jennit07 committed:
fix readme typos
few typos on some CHI to anonCHI examples. Added in a sample cohort example

Commit:
Re-build README.Rmd

Moohan commented on README.Rmd:
```diff
@@ -81,13 +86,47 @@ hscp_lookup <- slfhelper::partnerships
+set.seed(1)
+n <- 100
+# Make one column upi_number and one chi
+chi_cohort <- dplyr::tibble(
```
We don't need all this code in the Readme. The `chi_cohort` object already exists once the package is loaded

Moohan commented on README.Rmd:
```diff
@@ -59,6 +59,11 @@ indiv_1718 <- read_slf_individual("1718", partnerships = "S37000001") # Aberdeen
 # This can be a single recid or multiple by supplying a vector e.g. c(...)
 ep_1718 <- read_slf_episode("1718", recid = c("01B", "GLS"))
+
+#single recid filtering
```
Space after comment :p

(just run `{styler}`)

Moohan commented on README.Rmd:
```diff
@@ -81,13 +86,47 @@ hscp_lookup <- slfhelper::partnerships
+
+# Add anon_chi to cohort sample
+chi_cohort <- chi_cohort %>% 
   get_anon_chi(chi_var = "upi_number")
```
The `chi_var = "upi_number"` bit might need updating depending on what the CHI variable is called in the in-built `chi_cohort`

Moohan reviewed (changes requested):
A couple of minor changes - I wouldn't have mentioned the little ones if it wasn't that the big block of code needs removing anyway!

Could you also 'up' the years to be more recent, please? Maybe use 1920 instead of 1718.

Moohan committed:
Remove `chi_cohort` code block

This isn't needed in the Readme, as the `chi_cohort` object is already generated as part of the package.

Moohan committed:
style with `{styler}`

Commit:
Re-build README.Rmd
